var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -53,5 +58,3 @@ class Neovim(CMakePackage):
     depends_on('libluv@1.30.0:', type='link', when='@0.4:,stable')
     depends_on('tree-sitter', when='@0.5:')
 
-    def cmake_args(self):
-        return ['-DPREFER_LUA=ON']
```

Comment:
Probably, you wanted to add something like

```python
def cmake_args(self):
  return [self.define_from_variant('PREFER_LUA', 'use_lua')]
```

Reply:
If you prefer, I can add something like that back in.  At the moment I took it out because the configuration of neovim automatically picks the available lua, and currently no spec can have more than one implementation of lua so it always autodetects the right one.

Reply:
ah ok, I was not aware of that. I said that because I've seen the variant above 😉